accounts/manager.go

```diff
@@ -133,7 +134,7 @@ func NewManager(keydir string, scryptN, scryptP int, wantCacheDB bool) (*Manager
 }
 
 func (am *Manager) BuildIndexDB() []error {
-	return am.ac.Syncfs2db(time.Now().Add(-60*24*7*30*120*time.Minute)) // arbitrarily long "last updated"
+	return am.ac.Syncfs2db(time.Now().Add(-60 * 24 * 7 * 30 * 120 * time.Minute)) // arbitrarily long "last updated"
```

Comment:
Just in case we find many uses of time.Minute, later on we could do something like
```go
var (
    hour  = 60 * time.Minute
    day   = 24 * hour
    week  = 7 * day
    month = 4 * week
    year  = 12 * month
)
...
return am.ac.Syncfs2db(time.Now().Add(-1 * 10 * year))
```